vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow-integration-tests/src/main/java/com/vaadin/flow/component/menubar/tests/MenuBarTestPage.java

```diff
@@ -233,17 +234,38 @@ public MenuBarTestPage() {
                 });
         setUnsetSubItemClassNameButton.setId("set-unset-sub-item-class-name");
 
+        NativeButton setItem2ClassNameButton = new NativeButton(
+                "set item 2 class", e -> {
+                    item2.setClassName(MENU_ITEM_FIRST_CLASS_NAME);
+                });
+        setItem2ClassNameButton.setId("set-item2-class-name");
+
+        NativeButton removeItem2ClassNameButton = new NativeButton(
+                "remove item 2 class", e -> {
+                    item2.removeClassName(MENU_ITEM_FIRST_CLASS_NAME);
+                });
```

Comment:
This only works if the class name for "item 2" is "menu-item-first-class-name". I spent some time wondering why the button didn't work for me but it was because I had already clicked the "change item 2 class" button

Reply:
Changed to set to `null` instead.